app/views/layouts/email.html.erb

```diff
@@ -1,44 +1,21 @@
 <!doctype html>
 <html xmlns="http://www.w3.org/1999/xhtml">
   <head>
+    <meta name="viewport" content="width=device-width" />
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-    <title><%= yield :title %></title>
-
-    <style type="text/css">
-      .ReadMsgBody {width: 100%; background-color: #ffffff;}
-      .ExternalClass {width: 100%; background-color: #ffffff;}
-      body {width: 100%; background-color: #ffffff; margin:0; padding:0; -webkit-font-smoothing: antialiased;font-family: Arial, Helvetica, sans-serif}
-      table {border-collapse: collapse;}
 
-      @media only screen and (max-width: 640px)  {
-        body[yahoo] .deviceWidth {width:440px!important; padding:0;}
-        body[yahoo] .center {text-align: center!important;}
-      }
-
-      @media only screen and (max-width: 479px) {
-        body[yahoo] .deviceWidth {width:280px!important; padding:0;}
-        body[yahoo] .center {text-align: center!important;}
-      }
+    <title><%= yield :title %></title>
+    <style>
+      <% # Inject content of `email.scss`. Styles will be inlined by Roadie. %>
+      <%= Rails.application.assets['email.scss'].to_s %>
```

Comment:
This does not work with `assets.compile = false`, so need another way to inline css.